documentation/WhitePapers/SustainabilityUseCasesAndLandscape2023.md

```diff
@@ -2,6 +2,8 @@
 
 This captures the known and ongoing sustainability efforts within the cloud native landscape as well as identifies challenge areas.
 
+All relevant wordings can be found explained here in the [glossary](https://tag-env-sustainability.cncf.io/glossary/).
+
```

Comment:
Nice way to tie things together!  Do we want to add something to the effect of "if there is a wording you think needs added, feel free to submit a PR to include it?"  Or something along those lines?